web/crux-ui/src/validations/container.ts

```diff
@@ -207,7 +207,8 @@ const createOverlapTest = (
   )
 
 const portConfigRule = yup.mixed().when('portRanges', portRanges => {
-  if (!portRanges) {
+  // TODO(amorfevo): quickfixed, but why are we able to come here with such strange values at all?
+  if (!portRanges || !portRanges[0] || portRanges[0].length === 0) {
```

Comment:
`!portRanges?.[0]?.length` but summon js ninjas here @robot9706 @m8vago

Reply:
Yes, it's shorter and the same. I apply it.

Reply:
Or it could also be `!portRanges || Array.isArray(portRanges)` since it a portRange can't be an array.

Reply:
@robot9706 love that one, more readable